packages/shared/friends/sagas.ts

```diff
@@ -236,11 +248,16 @@ function* configureMatrixClient(action: SetMatrixClient) {
       recipient: message.sender === ownId ? senderUserId : identity.address
     }
 
+    let userProfile = getProfile(store.getState(), senderUserId)
+    if (!userProfile || !isAddedToCatalog(store.getState(), senderUserId)) {
+      await ensureFriendProfile(senderUserId)
```

Comment:
1) Shouldn't we send the profile to the renderer after this (as done in the previous case)?
2) If so, is there a way we can extract these two use cases of checking profile, catalog and send to renderer?

Reply:
The function `ensureFriendProfile` sends the profile to renderer (part of the sagas dependency tree)

Reply:
Got it! 👍🏻